### DISCUSSION TITLE
2023년 09일 21일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#80] [General]
# 2023년 09일 21일

⚖️ 공부한 것, 일기를 적는 개인 위키입니다 인덱스 페이지: [ wiki-index ] 이번 달의 TIL: [ 2023-09|2023년 9월의 TIL✨ ] 현재 자주 쓰는 글들: [ wooteco|우아한테크코스 ], [ shook|S-HOOK 개발기…

https://vero.wiki/2023-09-21/

**Comment by hong-sile** [comment]
아니 이제야 처음 남아봤다니 놀랍네요

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋㅋㅋ 이게 맞지...
